src/app.rs

```diff
@@ -9,12 +9,12 @@ use core::Named;
 use ecs::common::Errors;
 use ecs::prelude::{Component, World};
 use error::{Error, Result};
-use game_data::DataInit;
+use DataInit;
```

Comment:
pedantic: rustfmt should reorder this to the bottom iirc.

super pedantic (you can ignore this one -- it's not done in this repo): I normally split import groups into std, all other crates, and current crate, separated by an empty line.

```rust
use std::marker::PhantomData;

use not_this_crate::TypeA;
use another_crate::TypeB;

use local_module::TypeC;
```